_docs/Threat-Model.md

```diff
@@ -27,38 +27,38 @@ We distinguish between four attacker types:
+  2. Outside attacker who can eavesdrop traffic of a Qabel Drop server,
+  3. Outside attacker who can eavesdrop traffic of a Qabel Block server, 
   4. Outside attacker who can eavesdrop traffic of the public Internet (e.g., *DE-CIX Frankfurt*).
 
 ![Attacker Types](/images/attackerTypes.png)
 
 ## Capabilities of Attackers
 ### 0. Everyone
-The stored encrypted data is publicly available. Thus everybody knows the number and sizes of all files and the estimated number of all folders of every prefix.
+Writing and reading of drop messages is not restricted. Currently downloading from the Qabel Block server is restricted to authenticated accounts.
 
 ### 1. Contact
 A user *B* learns one identity (drop ID, public key and alias) of a user *A* during the contact. When receiving a share user *B* learns the directory structure of the shared directory and its files or the shared file. This implies that *B* learns the name of *A's* prefix the share is stored in.
 
 ### 2. Qabel User 
-An attacker has no advanced capabilities regarding security by being a Qabel user.
+The stored encrypted data is accessible for all Qabel accounts. Thus every user knows the number and sizes of all files and the estimated number of all folders of every prefix.
 
 ### 3. Qabel Servers
-Since a user has to be authenticated to be able to upload files on the server the provider knows the prefixes of each account. Due to this it is able to monitor all file writes and can match them to the registered user. During the creation of a folder it can observe in which parent folder it is created. Thereby it can reconstruct the directory tree of *A*. As soon as a user requests (meta) files in the order of the (sub-)directory tree, *O* can assume that *A* shared the (sub-)directory with the user.
+Since a user has to be authenticated to be able to upload files to the server the provider knows the prefixes of each account. Due to this it is able to monitor all file writes and can match them to the registered account. During the creation of a folder it can observe in which parent folder it is created. Thereby it can reconstruct the directory tree of *A*. As soon as an account requests (meta) files in the order of the (sub-)directory tree, *O* can assume that *A* shared the (sub-)directory with the an identity connected to this account.
 
-To prevent Qabel from being used as an illegal file distribution platform currently a user also needs to be authenticated for downloading a file. Thus the accounts of uploaders and downloaders can be linked. Since downloading of files can be conducted without possessing the respective file key this link can be faked.
+To prevent Qabel from being used as an illegal file distribution platform currently an account also needs to be authenticated for downloading a file. Thus the accounts of uploaders and downloaders can be linked. Since downloading of files can be conducted without possessing the respective file key this link can be forged.
```

Comment:
To prevent this, the download traffic is accounted to the owner of the prefix, but no authentication is required. This is fine because the file names are not publicly known and can be changed by the owner.